src/routes/patterns/components/Buttons.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { Button, ButtonGroup, Card } from 'components/baseline';
 
 const Buttons = () => (
-  <Card>
+  <Card className='Buttons'>
```

Comment:
I have it installed in my text editor. I've been using it already, but what your saying is we should enforce it?